.github/workflows/build-apko-image.yml

```
@@ -0,0 +1,28 @@
+name: Build and upload all apko container images
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - "release/**"
+      - "feat/reproducible-builds-ko-nogcp"
```

I think this could be allowed on any branch. Some devs build microservices on their feature branch during testing. The same applied to all workflows.

---

The initial ideas was to prevent unnecessary re-building without changes to the image definitions.
Since all the `apko` image definitions should reside in `image/apko` anyways, we could maybe remove the branch filtering and add path filtering for the said folder?

This could look like this:
```suggestion
          paths:
            - 'image/apko/*.yaml'

```

---

I slightly misread the code and thought the branch filter was on the `workflow_dispatch`. Adding an additional path filter for the apko yamls on this workflow sounds like a good idea though :)